<ui version="4.0" >
 <class>TrackEditorClass</class>
 <widget class="QWidget" name="TrackEditorClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TrackEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="TrackEditorRenderArea" native="1" name="trackEditorRenderArea" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>320</width>
         <height>320</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <property name="sizeConstraint" >
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Glätten</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6" >
          <item>
           <widget class="QDoubleSpinBox" name="thresholdSpinBox" >
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="suffix" >
             <string> m</string>
            </property>
            <property name="decimals" >
             <number>1</number>
            </property>
            <property name="singleStep" >
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="weedingTypeComboBox" >
            <property name="sizeAdjustPolicy" >
             <enum>QComboBox::AdjustToMinimumContentsLength</enum>
            </property>
            <item>
             <property name="text" >
              <string>3D (Douglas-Peucker)</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>2D (Douglas-Peucker)</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Höhe (Douglas-Peucker)</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Höhe (Schwellwert)</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>81</width>
              <height>128</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Track-Info</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>Strecke [km]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="trackDistanceLabel" >
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11" >
            <property name="text" >
             <string># Punkte</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="trackPointCountLabel" >
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>Anstiege [m]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ascentLabel" >
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Abstiege [m]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="descentLabel" >
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_12" >
            <property name="text" >
             <string>min. Höhe [m]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="minEleLabel" >
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13" >
            <property name="text" >
             <string>max. Höhe [m]</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="maxEleLabel" >
            <property name="frameShape" >
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow" >
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="alignment" >
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3" >
            <property name="orientation" >
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType" >
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>76</width>
              <height>128</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ElevationProfileRenderArea" name="elevationProfileRenderArea" >
     <property name="mouseTracking" >
      <bool>true</bool>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>trackEditorRenderArea</zorder>
  <zorder>elevationProfileRenderArea</zorder>
  <zorder>trackEditorRenderArea</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TrackEditorRenderArea</class>
   <extends>QWidget</extends>
   <header>TrackEditorRenderArea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ElevationProfileRenderArea</class>
   <extends>QFrame</extends>
   <header>ElevationProfileRenderArea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
